Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py

```diff
@@ -533,6 +534,49 @@ def parse_asset_data(self, issue: Dict[str, Any],
             changed = True
         return assets, ml_feature_list, changed
 
+    def get_exposures_ips(self, params):
```

Comment:
The Exposures API is being deprecated, so we shouldn't include this in the new version.